lib/date.js

```diff
@@ -0,0 +1,22 @@
+import moment from 'moment';
+
+export function exhibitionPeriod(startAt, endAt) {
```

Comment:
wot? not `ausstellungssauer` - [shocking](https://github.com/artsy/eigen/issues/16)